src/wp-includes/media.php

```diff
@@ -2050,6 +2074,9 @@ function wp_img_tag_add_loading_optimization_attrs( $image, $context ) {
  * @return string Converted `img` tag with `decoding` attribute added.
  */
 function wp_img_tag_add_decoding_attr( $image, $context ) {
+	# see https://core.trac.wordpress.org/ticket/58892
+	_deprecated_function( __FUNCTION__, '6.4.0', 'wp_get_loading_optimization_attributes()' );
```

Comment:
Add missing `@deprecated` annotation for the function.

`* @deprecated 6.4.0 Use wp_get_loading_optimization_attributes()`